src/dlgProfilePreferences.cpp

```diff
@@ -62,16 +62,11 @@ dlgProfilePreferences::dlgProfilePreferences(QWidget* pF, Host* pH) : QDialog(pF
     mFORCE_MXP_NEGOTIATION_OFF->setChecked(mpHost->mFORCE_MXP_NEGOTIATION_OFF);
     mMapperUseAntiAlias->setChecked(mpHost->mMapperUseAntiAlias);
     acceptServerGUI->setChecked(mpHost->mAcceptServerGUI);
-    QString nick = tr("Mudlet%1").arg(QString::number(rand() % 10000));
-    QFile file(QDir::homePath() + "/.config/mudlet/irc_nick");
-    file.open(QIODevice::ReadOnly);
-    QDataStream ifs(&file);
-    ifs >> nick;
-    file.close();
-    if (nick.isEmpty()) {
```

Comment:
Is this 'generate a random number to put on the end of "Mudlet"' code redone somewhere _else_ if there is not a global OR a per-profile Nickname setting stored?